domain/consensus/ruleerrors/rule_error.go

```diff
@@ -236,6 +236,9 @@ var (
 
 	//ErrPruningPointViolation indicates that the pruning point isn't in the block past.
 	ErrPruningPointViolation = newRuleError("ErrPruningPointViolation")
+
+	//ErrDelayedBlock indicates that timeStamp is in the future.
```

Comment:
Please fix comment to "ErrDelayedBlock indicates that the block timestamp is too much in the future."